clang/include/clang/Basic/Attr.td

```diff
@@ -2582,8 +2586,8 @@ def NoDeref : TypeAttr {
 // intel::reqd_work_group_size spelling.
 def ReqdWorkGroupSize : InheritableAttr {
   let Spellings = [GNU<"reqd_work_group_size">,
-                   CXX11<"intel","reqd_work_group_size">, 
-                   CXX11<"cl","reqd_work_group_size">]; 
+                   CXX11<"intel","reqd_work_group_size">,
+                   CXX11<"cl","reqd_work_group_size">];
```

Comment:
Seems like unrelated change.

Reply:
Ah, shoot... my editor is configured to remove trailing WS on save and must have picked it up.  @bader, you resolved this conversation, are you OK with it, or would you like me to revert this?

Reply:
I'm okay with it.